viewer/src/views/threejs/sector/createThreeJsSectorNode.ts

```diff
@@ -37,13 +37,6 @@ export async function createThreeJsSectorNode(model: SectorModel): Promise<Secto
     rootGroup.needsRedraw = true;
   };
 
-  // Sync high- and low-detail geometry
-  const [discardSectorFinal, consumeSectorFinal, consumeSectorQuadsFinal] = createSyncedConsumeAndDiscard(
```

Comment:
Should we even bother to keep the `createSyncedConsumeAndDiscard`-implementation around?

Reply:
Nope. I will remove that now.